README.md

```diff
@@ -18,7 +18,6 @@ Currently supported cloud flavours are:
 Following features and tools are supported:
 * 🐟 Fish Shell
 * 📷 AzCopy
-* 🔐 Bitwarden
```

Comment:
You maybe should have put the removal of this feature in an own commit.

Reply:
Yeah, you're right. Tim also already mentioned that. I'll rebase the change into a single commit once @rexhinvorpsi gave his okay.

Reply:
Done